apps/api/src/seeds/seed-test-data.ts

```diff
@@ -53,7 +55,7 @@ export const algorandAccountFactory = Factory.define<AlgorandAccount>(
     (creationTransaction) => creationTransaction.id
   )
   .attr('encryptedKey', ['mnemonic', 'passphrase'], (mnemonic, passphrase) =>
-    encrypt(mnemonic, passphrase, TEST_APP_SECRET)
```

Comment:
Good catch.  I was trying to simplify certain things by introducing `TEST_APP_SECRET`, but seeing it in someone elses PR I can see how that would cause problems.

Reply:
@tj-boyle Just a heads up, eventually this file might be shared between api and scribe codebases, which means it will need to be moved out of the api app, which means that we'll have to refactor out the use of `Configuration.secret`.  When/if that eventually happens I guess we'd just modify the factory to accept the secret as a parameter.

Reply:
yup! good callout